src/plugins/syscalls/syscalls.cpp

```diff
@@ -106,32 +106,63 @@
 #include <glib.h>
 #include <inttypes.h>
 #include "syscalls.h"
+#include "scproto.h"
 
 static event_response_t cb(drakvuf_t drakvuf, drakvuf_trap_info_t *info) {
```

Comment:
This function is being used a callback for both Linux and Windows. This PR only adds argument extraction for Windows, so I think making two separate callback functions would be better for now.